src/attribute-editor/additional-info.tsx

```diff
@@ -1,12 +1,18 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import React from 'react';
+import LiveRegion from '../internal/components/live-region';
 import styles from './styles.css.js';
 
 interface AdditionalInfoProps {
   children: React.ReactNode;
+  id: string;
 }
 
-export const AdditionalInfo = ({ children }: AdditionalInfoProps) => (
-  <div className={styles['additional-info']}>{children}</div>
+export const AdditionalInfo = ({ children, id }: AdditionalInfoProps) => (
+  <LiveRegion visible={true}>
+    <div id={id} className={styles['additional-info']}>
```

Comment:
Both `id` and `className` can be assigned to the LiveRegion, the extra div is not necessary.

Reply:
It doesn't quite work for a few reasons:

1. We're technically not supposed to use `id` and `className` on Cloudscape components
2. The styles get messed up because `LiveRegion` wraps the visible children in a span, but it needs to be a div.
3. `LiveRegion` applies the `restProps` (`id`, `className`, ...) to the screen reader only part, not the visible part.

Reply:
I had to move things around a little bit more because the Tag editor was causing invalid HTML (divs inside spans). I've extended LiveRegion to specify an optional tag name.